referenceframe/urdf/testfiles/example_gantry.urdf

```diff
@@ -16,7 +16,7 @@
     <child link="gantry_carriage"/>
     <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.15"/>
     <axis xyz="0 1 0"/>
-    <limit lower="-0.1" upper="2.1" />
+    <limit effort="150" lower="-0.1" upper="2.1" velocity="3.141592" />
```

Comment:
@raybjork IDK what these effort & velocity values should be for this file

Reply:
I think lets just remove them from this file.  I don't think we ever want this file to be used as a urdf for something because it doesn't actually map to off the shelf hardware, so no sense making up bogus numbers I think?  Could also go the other way on it if you think theres a reason to

Reply:
Fair enough but if we do that I wonder if we should change this file to not have the urdf extension given that it is not valid urdf.

Should I just change it to `example_gantry.xml`?

Reply:
Yeah I think that makes sense.  Its only used in one test anyway.  Maybe you can add a comment to the top of the file explaining that its not valid urdf but used for tests?